src/main/java/com/max/grpc/orders/server/Main.java

```diff
@@ -0,0 +1,13 @@
+
+package com.max.grpc.orders.server;
+
+import com.max.grpc.orders.server.food.FoodManager;
+
+public class Main {
+    public static void main(String[] args) {
+        var foodManager = new FoodManager();
+        var server = new CafeServer(8000, foodManager);

```

Comment:
What is 8000 mean?

Reply:
Port

Reply:
Oh, I see, thanks for the explanation. But the code should be descriptive enough, to do not ask developers about such things.

Reply:
Ok, please suggest how to make the code more descriptive? Use builder?

Reply:
You can make a constant and call it SERVER_PORT, and simply use it. Or even more, you can extract it to the property files, to make your server more configurable.

Reply:
Added loading server port from `resources/server.properties`